ui/src/app/components/icons.tsx

```diff
@@ -14,11 +14,14 @@ import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
 
 import colors, {addOpacity} from 'app/styles/colors';
 
-import {ReactComponent as registeredTierBadge} from 'assets/icons/registered-tier-badge.svg';
```

Comment:
Why are these imported differently?

Reply:
Webpack requires images to be treated this way.
https://create-react-app.dev/docs/adding-images-fonts-and-files/

Reply:
Thanks!  What's confusing to me is that the other 2 (arrow and googleCloudLogo) work differently.

Reply:
Oh! I'm not 100% sure why it's like that. Maybe a bad merge on my part. The way that arrow and googleCloudLogo are should work for the tier badges.